### DISCUSSION TITLE
Group Chat

### DISCUSSION THREAD

**Discussion by pnxzadrot** [chatwoot/chatwoot#6991] [Q&A]
Good day!

Can you help me, because I don't understand, can Chatwoot create a group chat with  some client and one or zero agent?

**Comment by vishnu-narayanan** [comment]
Converting this to a discussion as this is not an issue.

**Comment by vishnu-narayanan** [comment]
@pnxzadrot Chatwoot creates a single conversation against a customer interaction. All agents in an inbox can see/interact with this conversation. Conversation can only be assigned to a single agent, while multiple agents can be added as conversation participants. 

Agents can also chat/collaborate against themselves using private notes in a conversation.

ref: https://www.chatwoot.com/docs/user-guide/conversation-workflow/#private-chat-with-agents

**Reply by pnxzadrot** [reply]
Thanks by answer, but I ask about one chat for several client, not agent. How I see, chat create for client and several agent can answer. But how create chat for several client, or one client and add anothers  clients?

**Reply by vishnu-narayanan** [reply]
@pnxzadrot Chatwoot does not support a conversation with multiple customers at the moment.
